### ISSUE TITLE
module Earmark is not available

### ISSUE THREAD

**Issue by RobStallion** [healthlocker/healthlocker#887]
function Earmark.as_html!/1 is undefined (module Earmark is not available)

![image](https://user-images.githubusercontent.com/15571853/27842758-b7b22c20-6104-11e7-97de-750674642c2a.png)

**Comment by RobStallion** [comment]
Got the following error message when I was attempting to run a build using edeliver..
![image](https://user-images.githubusercontent.com/15571853/27873421-f8a2114c-61a3-11e7-8337-a148efbe118a.png)

```
==> One or more direct or transitive dependencies are missing from
    :applications or :included_applications, they will not be included
    in the release:

    :bamboo_smtp
    :earmark
    :ecto_factory
    :elixir_make
    :wallaby

    This can cause your application to fail at runtime. If you are sure
    that this is not an issue, you may ignore this warning.
```

It looks like earmark will not be used in the build which is why it was not working in the staging env.

Going to try adding earmark to the applications for the next build

**Comment by RobStallion** [comment]
adding earmark to list of project applications in order to try and resolve the the issue listed above

**Comment by RobStallion** [comment]
fixed by #889
